src/app/f2f/controllers/checkDetails.js

```diff
@@ -105,19 +106,18 @@ class CheckDetailsController extends DateController {
           idHasExpiryDate = req.form.values.idHasExpiryDate
           expiryDate = req.form.values.euPhotocardDlExpiryDate;
           country = req.form.values.euDrivingLicenceCountrySelector;
-          address = req.form.values.euDrivingLicenceAddressCheck
+          address = req.form.values.euPhotocardDlAddressCheck
```

Comment:
Out of interest how come we renamed these?

Reply:
This is to align with the name that is mapped to photoIdChoice, so that when the first part of AddressCheck is interpolated in checkDetails controller it grabs the right value from the yaml file